tests/test_examples_jslib.py

```diff
@@ -0,0 +1,82 @@
+import json
+from pathlib import Path
+
+from tests.conftest import FlowHeaterLayer
+from tests.util import execute
+from tests.preprocessor import preprocess_javascript
+
+basedir = Path("examples/06-javascript-libs")
+
+
+def test_modhello(fh_http: FlowHeaterLayer):
+
+    # ``fh-core`` doesn't know how to interpret
+    # the non-standard ``@fh:include()`` yet.
+    # So, let's resolve this in user space.
+    jsfile = basedir / "01-basic-module/userspace.js"
```

Comment:
just really recognizing this for the first time. Is this the new `os.path.join()` shortcut? I mean using a slash in string concatenation?

Reply:
Exactly. You can do this through operator overloading, but also a lot more on the OO-API.
See https://docs.python.org/3/library/pathlib.html.